### DISCUSSION TITLE
Changing the baseurl later on

### DISCUSSION THREAD

**Discussion by MTRNord** [klis87/redux-saga-requests#552] [Q&A]
Hi,

I am trying to build a matrix client and since the servers can be selfhosted the URL of my api is only known as soon as the user inputs data and this data might change before the user is fully logged in because you put in your url, the client requests a file to find the real url and the user still may change this url then. Only after the user pressed the login button the url is known to be "stable":

Is this possible using redux-requests? :) I wasnt able to find a obvious way to do that but I am also new to redux workflow so I might be missing the obvious way to do this :)

Oh I am using the axios driver for this.

**Comment by klis87** [comment]
You can set `baseUrl` for each request independently, because you use `axios`.
